### PR TITLE
Add prefill backend documentation

### LINKED ISSUE #622: Update documentation for prefill backends
**Tasks**

- [x] Add (in Dutch) example form to show prefill (use KvK, since its free to get the API key). See [here](https://open-forms.readthedocs.io/en/latest/manual/forms/examples/autofill_address.html) for example)
- [x] Document (in English) how to configure pre-fill backend: KvK
- [x] Document (in English) how to configure pre-fill backend: StUF-BG
- [x] Document (in English) how to configure pre-fill backend: Haal Centraal Bevragen personen

See [here](https://open-forms.readthedocs.io/en/latest/configuration/prefill/bag.html) for examples

### PR DESCRIPTION
**Pull request by shea-maykinmedia**
Fixes #622

### CODE REVIEW

File: docs/configuration/prefill/haal_centraal.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.. _configuration_prefill_haal_centraal:
+
+=============
+Haal Centraal
+=============
+
+Haal Centraal is an API that allows you to retrieve personal information about the person
+filling out the form.
+
+1. Obtain a `Haal Centraal API key`_ from Kadaster.
```
Comment (shea-maykinmedia) [inline docs/configuration/prefill/haal_centraal.rst:10]: Is this where a user would get a Haal Centraal API key from?

File: docs/configuration/prefill/stuf_bg.rst [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+=======
+StUF-BG
+=======
+
+StUF-BG (StUF Basis Gegegevens) is a message standard that allows retrieving of
+personal information by API from municipalities and government organisations.
+Open Forms can be configured to use this service to prefill personal data of
+the person filling out the form.
+
+1. You will need to have a contract with a municipality or government organisation to use this plugin.
```
Comment (shea-maykinmedia) [inline docs/configuration/prefill/stuf_bg.rst:12]: Is this correct?  Is there a specific organisation they would need a contract with?

File: docs/manual/forms/examples/prefill.rst [hunk 3]
```diff
@@ -0,0 +1,48 @@
+=======
+Prefill
+=======
+
+In dit voorbeeld tonen we hoe u een prefill kunnen gebruiken.  Prefill is een optie zodat
+een formulier prefilled met informatie is.  Met dit voorbeeld tonen we hoe u de
```
Comment (shea-maykinmedia) [inline docs/manual/forms/examples/prefill.rst:6]: Is there a dutch word for `prefilled`?

File: docs/configuration/prefill/stuf_bg.rst [hunk 4]
```diff
@@ -0,0 +1,26 @@
+.. _configuration_prefill_stuf_bg:
+
+=======
+StUF-BG
+=======
+
+StUF-BG (StUF Basis Gegegevens) is a message standard that allows retrieving of
```
Comment (shea-maykinmedia) [inline docs/configuration/prefill/stuf_bg.rst:7]: Should this link to gemmaonline.nl?

**Review by joeribekker — COMMENTED** [review]
I'll make the necessery changes.

File: docs/manual/forms/examples/prefill.rst (hunk 3 above)
Comment (joeribekker) [reply docs/manual/forms/examples/prefill.rst:6]: I use prefilled, but technically it's "voorinvullen"

File: docs/configuration/prefill/stuf_bg.rst (hunk 4 above)
Comment (joeribekker) [reply docs/configuration/prefill/stuf_bg.rst:7]: nah its fine.

**Review by joeribekker — APPROVED** [review]

**Pull request merged**